.github/workflows/create_version_bump_pr.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: .\tracer\build.ps1 UpdateChangeLog
 
       - name: "Bump Version"
-        run: .\tracer\build.ps1 UpdateIntegrationDefinitions UpdateVersion UpdateMsiContents
+        run: .\tracer\build.ps1 UpdateVersion BuildTracerHome UpdateMsiContents
```

Comment:
If I understand correctly, you're invoking a re-build of the Tracer to update the source-generated integration definitions, is that right? Nowadays there's nothing inherently version-related in the generated integration definitions (after we switched to passing integrations via PInvoke) and with the source generator the file should be generated immediately if there's a change so I wonder if we even need this call. Do you think we can remove this?

Reply:
Hmmm. I guess that's true, yeah 🤔 And given we have the validation that the persisted files are already correct, there's nothing to "mop up" I don't think

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/2305/commits/bf9e2270153e05fe6073a367bff910088a5a92dd